bftengine/src/bcstatetransfer/RVBManager.cpp

```diff
@@ -770,45 +775,63 @@ void RVBManager::reportLastAgreedPrunableBlockId(uint64_t lastAgreedPrunableBloc
   }
 
   uint64_t current_rvb_id = start_rvb_id;
-  int32_t num_digests_added{0};
-  while (current_rvb_id <= lastAgreedPrunableBlockId) {
-    Digest digest;
-    if (!pruned_blocks_digests_.empty() && (current_rvb_id <= pruned_blocks_digests_.back().first)) {
+  // Avoid fetching already existing digests of to-be-pruned blocks
+  if (not pruned_blocks_digests_.empty()) {
```

Comment:
You should use the same code style: if you use 'not' and 'and', use it everywhere. Even in the code you added by yourself, it is not consistent.